client/client-multi/client-objective-c/Kaa/log/AbstractLogCollector.h

```diff
@@ -39,8 +39,10 @@
 
 - (void)uploadIfNeeded;
 
-- (void)addDeliveryRunner:(BucketRunner *)runner bucketInfo:(BucketInfo *)bucketInfo;
+- (void)addDeliveryRunner:(BucketRunner *)runner byBucketInfoKey:(NSNumber *)bucketInfoKey;
 
 - (void)notifyOnSuccessDeliveryRunnersWithBucketInfo:(BucketInfo *)bucketInfo;
 
+- (NSMutableDictionary *)getDeliveryRunnerDictionary;
```

Comment:
Yes, this method had been used at debug process and hasn't been removed after bug fixing. Removed